helm/templates/api/api-deployment.yaml

```
@@ -115,8 +115,10 @@ spec:
               containerPort: {{ .Values.api.http.services.core.http.port | default "18093" }}
             {{- end }}
           env:
-            - name: upgrade.mode
+            {{- if .Values.api.upgrader }}
+            - name: "upgrade.mode"
               value: "false"
+            {{- end }}
```

```suggestion
            {{- if .Values.api.upgrader }}
            - name: "upgrade.mode"
              value: {{ .Values.api.upgrader }}
            {{- end }}
```
This means that the upgrader setup can not be true :thinking:

---

on the mAPI container, this value should be either false (in case of the job is enabled) or missing to always execute it.